plugins/jquery-filters.php

```diff
@@ -14,16 +14,20 @@
 $sites = jquery_sites();
 $options = array_merge( $options, $sites[ JQUERY_LIVE_SITE ]['options'] );
 foreach ( $options as $option => $value ) {
-	if ( $option === 'stylesheet' || $option === 'template' ) {
-		// Don't mess with themes for now.
-		continue;
-	}
-	if ( $option === 'active_plugins' ) {
-		// Deprecated
-		// In production, Puppet manages activation of per-site plugins.
-		// Locally, global plugins are enabled via mu-plugins/_loader.php,
-		// and per-site plugins are activated by the imported database.
-		continue;
+	// Skip these in production, where they are managed by puppet.
+	// Staging should be allowed to set them for testing.
+	if ( !JQUERY_STAGING ) {
+		if ( $option === 'stylesheet' || $option === 'template' ) {
+			// Don't mess with themes for now.
+			continue;
+		}
+		if ( $option === 'active_plugins' ) {
+			// Deprecated
+			// In production, Puppet manages activation of per-site plugins.
+			// Locally, global plugins are enabled via mu-plugins/_loader.php,
+			// and per-site plugins are activated by the imported database.
```

Comment:
Should we remove the "Deprecated" and "Locally" phrasing here?

Reply:
I agree with removing "Deprecated". "Locally" is still accurate tho, right?

Reply:
No, because this code would no longer run locally. We're specifically making it so that per-site config is now determined by sites.php instead of by what is in the database.

Reply:
I'm not sure I follow. "Locally" here describes the alternate code path, which is sites.php.

Reply:
I see